playbooks/roles/ocp-cost-management/tasks/main.yml

```diff
@@ -0,0 +1,121 @@
+---
+# tasks file for playbooks/roles/ocp-cost-management
+
+- name: Check if cluster operators and nodes are healthy
+  include_role:
+    name: check-cluster-health
+
+- name: Clean up
+  include_tasks: "{{ role_path }}/tasks/cmo-cleanup.yml"
+  when: cost_management_cleanup
+
+- name: cost management operator setup
+  include_tasks: "{{ role_path }}/tasks/cmo-setup.yml"
+  when: cost_management_setup
+
+- name: cost management operator test
+  include_tasks: "{{ role_path }}/tasks/cmo-test.yml"
+  when: cost_management_test
+
+- name: Create working directory
+  file:
+    state: "directory"
+    path: "{{ cost_management_directory }}"
+
+- name: Setup custom catalog source
+  block:
+  - name: Include the global-secret-update role
+    include_role:
+        name: global-secret-update
+
+  - name: Set fact variable for CatalogSource name
+    set_fact:
+      cost_management_catalogsource: "qe-app-registry"
+
+  - name: Include role to create ImageContentSourcePolicy and CatalogSource
+    include_role:
+      name: set-custom-catalogsource
+    vars:
+      custom_catalogsource_name: "{{ cost_management_catalogsource }}"
+      custom_catalogsource_display_name: "Cost Management Operator"
```

Comment:
Can you please use the same catalog source name as required for the tests, i.e., `my-operator-catalog`, so that we can use a single catalog source?

Reply:
I ment this catalog - https://github.com/Pavan-Sadhana3/ocp4-playbooks-extras/blob/8231b41ae6fca2f61835ca0b4e2ca2fa71efb03c/playbooks/roles/ocp-cost-management/tasks/cmo-test.yml#L24